### PR TITLE
Jgordon new tracking num branch

### PR DESCRIPTION
**Pull request by jathmel**
Add log to display error when shipping provider is missing

### CODE REVIEW

File: src/Services/ShipmentNotificationService.php [hunk 1]
```diff
@@ -489,8 +489,21 @@ public function prepareRequestDto(Order $order): RequestDTO
             'method' => __METHOD__
           ]
         );
-
-        $scacCode = $this->carrierScacRepository->findScacByCarrierId($plentymarketsShippingInformation->shippingServiceProvider->id);
+        if (empty($plentyMarketsShippingInformation->shippingServiceProvider->id)) {
```
Comment (jhoule86) [inline src/Services/ShipmentNotificationService.php:492]: if you're checking for "null or empty", you should do an `!isset(...)` check here as well.

File: resources/lang/en/logger.properties [hunk 2]
```diff
@@ -124,3 +124,4 @@ pendingOrderMayExist="Pending Order may exist"
 failedToFetchOrders="Failed to fetch orders from Wayfair"
 fetchingOrders="Fetching orders from Wayfair"
 incomingPurchaseOrder="Incoming purchase order"
+shippingRegisteredAlready="The shipment has already been registered"
```
Comment (jhoule86) [inline resources/lang/en/logger.properties:127]: I don't see where this is being used.

Comment (jathmel) [reply resources/lang/en/logger.properties:127]: It was a constant defined in `shipmentRegisterService` but had no data in the properties file so I added it

Comment (jhoule86) [reply resources/lang/en/logger.properties:127]: oh. nice.

**Pull request merged**